Makefile

```diff
@@ -210,43 +218,70 @@ deploy-tektoncd-dashboard: tektoncd-dashboard-helm
 add-bitnami-repo:
 	@helm repo add bitnami https://charts.bitnami.com/bitnami
 
-dev-dracon: deploy-elasticoperator deploy-arangodb-crds add-bitnami-repo
+deploy-cluster:
+	@scripts/kind-with-registry.sh
+
+install: deploy-cluster deploy-elasticoperator deploy-arangodb-crds add-bitnami-repo
 	@echo "fetching dependencies if needed"
 	@helm dependency build ./deploy/dracon/chart
-	@echo "deploying dracon in dev mode"
+	@echo "deploying dracon"
 	@helm upgrade dracon ./deploy/dracon/chart \
+	 	  --install \
+		  --values ./deploy/dracon/values/dev.yaml \
+		  --create-namespace \
+		  --set "image.registry=$(CONTAINER_REPO)" \
+		  --namespace $(DRACON_NS) \
+		  --version $(DRACON_VERSION) \
+		  --wait
+	@helm upgrade deduplication-db-migrations ./deploy/deduplication-db-migrations/chart \
```

Comment:
would add an echo before this to say "running migrations", just so if the script gets stuck there it's easier to debug